sonic-psud/scripts/psud

```diff
@@ -271,13 +339,17 @@ class DaemonPsud(daemon_base.DaemonBase):
             self.update_psu_data(psu_tbl)
             self._update_led_color(psu_tbl)
 
+            self.update_psu_chassis_info(chassis_tbl)
```

Comment:
Is it applicable only for chassis? Can it be under if (chassis) {... }

Reply:
Done.